test/controllers/ModulesSpec.scala

```diff
@@ -52,11 +63,10 @@ object ModulesSpec extends PlaySpecification with Mockito {
     }
   }
 
-  def createModules: (Modules, ModulesLookup, ModuleDao) = {
-    val lookup = mock[ModulesLookup]
-    val dao = mock[ModuleDao]
-    val modules = new Modules(lookup, dao)
+  def createModules(app: play.api.Application): (Modules, ModulesLookup, ModuleDao) = {
+    val lookup = app.injector.instanceOf[ModulesLookup]
```

Comment:
I thought we had the instance of app only because we needed it for the global state in previous versions of Play.

If the only thing we need is the default action builder, we could also mock only that method. Or we could inject `action: DefaultActionBuilder` directly. If that's too hard, we need to make it easier I think.